src/setup-gcloud.ts

```diff
@@ -36,7 +36,7 @@ const actionRef = process.env.GITHUB_ACTION_REF;
 export async function run(): Promise<void> {
   if (actionRef == 'main' || actionRef == 'master') {
     core.warning(
-      `google-github-actions/setup-gcloud is pinned at HEAD. We strongly ` +
+      `google-github-actions/setup-gcloud is effectively pinned to HEAD. We strongly ` +
```

Comment:
We should probably just say "@master" instead here.

Reply:
You can't since line 37 also covers `main`.

This was the smallest change I could make that was definitely not wrong.

I'm not claiming that the line below that says `@master` is right...

Reply:
Yea, but we know this particular action is pinned `@master` right now 😄 . We actually centralized this into a helper in https://github.com/google-github-actions/actions-utils/blob/f2e0cec9ce45d80c987943bdf3f5216734fa1f06/src/warnings.ts#L41, but there's some refactoring to do before we can bring that package into setup-gcloud right now.

If you wanted, you could grab the actual ref from `process.env.GITHUB_ACTION_REF`, which would theoretically handle either master or main.

Reply:
If you're ok with that change, I might be able to do it tonight.

Reply:
SGMT!